muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-management.js

```diff
@@ -108,6 +110,35 @@
     }
   });
   
+  
+  $.widget("custom.workspaceChatSettings", {
+    options: {
+      workspaceEntityId: null,
+    },
+    _create: function() {
+      var workspaceEntityId = this.options.workspaceEntityId;
+     mApi().chat.workspaceChatSettings.read(workspaceEntityId).callback($.proxy(function (err, workspaceChatSettings) {
+        if (err) { 
+          $('.notification-queue').notificationQueue('notification', 'error', err);
+          return;
+        }
+        var data = {};
+        if (workspaceChatSettings === null){
+          data.disabled_selected = "selected";
+        }
+        if (workspaceChatSettings && workspaceChatSettings.chatStatus === "ENABLED") {
+          data.enabled_selected = "selected";
+        }
+        if (workspaceChatSettings && workspaceChatSettings.chatStatus === "DISABLED"){
```

Comment:
These three if statements can be replaced simply as...

`if (workspaceChatSettings && workspaceChatSettings.chatStatus === "ENABLED") {`
  `data.enabled_selected = "selected";`
`} else {`
  `data.disabled_selected = "selected";`
`}`

....i.e. no need to do three separate `if` comparisons in a row when we can do just one.